test/euclidean.jl

```diff
@@ -147,4 +147,31 @@ include("utils.jl")
         @test ℍ^2 === Euclidean(2; field = ℍ)
         @test ℍ^(2, 3) === Euclidean(2, 3; field = ℍ)
     end
+
+    @testset "Embeddings into larger Euclidean Manifolds" begin
```

Comment:
I think it would be good to have a simple test for embedding a real Euclidean space into a complex Euclidean space.

Reply:
Good hint, I mixed up the fields (the embedded manifold is still of field type as the smaller one, not as the embedding). I added a test for that.

Reply:
Right, I mixed it up as well when I suggested this change. Thanks for fixing it :slightly_smiling_face: .